kbfstool/md_main.go

```diff
@@ -12,6 +12,7 @@ const mdUsageStr = `Usage:
 
 The possible subcommands are:
   dump		Dump metadata objects
+  check		Check metadata objects and their associated blocks for errors
```

Comment:
I was going to comment that `IsValidAndSigned` is already called by `MDOps`, but that wasn't quite true! Rebased this on top of https://github.com/keybase/kbfs/pull/263 , which makes that true, and added comments.